test/bdd/fixtures/oauth-clients/clients.json

```diff
@@ -4,7 +4,13 @@
     "redirect_uris": ["http://127.0.0.1/callback"],
     "grant_types": ["authorization_code"],
     "response_types": ["code"],
-    "scopes": ["openid","profile"],
+    "scopes": [
+      "openid",
+      "profile",
+      "https://api-gateway.trustbloc.local:5566/issuer/bank_issuer/v1.0",
+      "https://api-gateway.trustbloc.local:5566/issuer/i_myprofile_cmtr_p256_ldp/v1.0",
+      "https://api-gateway.trustbloc.local:5566/issuer/i_myprofile_ud_es256k_jwt/v1.0"
```

Comment:
as per our grooming discussion - wallet creates these scopes dynamically for now.
<ISSUER_IDENTIFIER> from issuer's OIDC server configuration

Reply:
This lines are required for bdd-test. for production admin team should provide a correct list of scopes in auth schema